README.md

```diff
@@ -1,3 +1,12 @@
+---
```

Comment:
why would running it through ipymd add this particular metadata?
ideally, running a standard Markdown file through ipymd (round-trip) should not change it at all. I can't remember if there's a test for that -- if not, we should add one.